Marlin/ultralcd_impl_HD44780.h

```diff
@@ -795,7 +795,9 @@ static void lcd_implementation_status_screen() {
 
   #endif // FILAMENT_LCD_DISPLAY && SDSUPPORT
 
-  lcd_print(lcd_status_message);
+  char status_message[LCD_WIDTH + 1];
+  strlcpy(status_message, lcd_status_message, sizeof(status_message));
```

Comment:
Rather than instantiate another variable, couldn't we just 
`lcd_status_message[LCD_WIDTH+1]=0;` ?

It looks like we're at the end of the function here, so I suspect there's no other use for the clipped text?  Or is it a global that may be used elsewhere?

Either way is fine, really..  Not like the new variable costs much.  Just a random thought.